src/core/test_helpers/kbn_server.ts

```diff
@@ -99,7 +98,6 @@ export function createRoot(settings = {}, cliArgs: Partial<CliArgs> = {}) {
  */
 export function createRootWithCorePlugins(settings = {}, cliArgs: Partial<CliArgs> = {}) {
   const DEFAULT_SETTINGS_WITH_CORE_PLUGINS = {
-    plugins: { scanDirs: [resolve(__dirname, '../../legacy/core_plugins')] },
```

Comment:
This must have been here for a long time, because `/src/legacy/core_plugins` was removed before the end of the KP migration 😅